Laba3.c

```diff
@@ -0,0 +1,93 @@
+#include <stdio.h>
+#include <stdlib.h>
+// For some reason Visual Studio wants scanf to be changed to scanf_s
+#define scanf scanf_s
+
+int main()
+{
+    int n1, n2, k, * arr1, * arr2, * arrd;
+    // Inputting array sizes
+    printf("Input n1: ");
+    scanf("%d", &n1);
+    printf("Input n2: ");
+    scanf("%d", &n2);
+    // Allocating memory for the arrays
+    arr1 = (int*)malloc(n1 * sizeof(int));
+    arr2 = (int*)malloc(n2 * sizeof(int));
+    // Allocating memory for resulting array with a size of the smallest array among these two above
+    if (n2 <= n1)
```

Comment:
Suggestion: simplify the expression: 

`arrd = (int*)malloc((n1 <= n2 ? n1 : n2) * sizeof(int));`